### PR TITLE
Add support for reading IrregularlySampledSignals in AsciiSignalIO

### PR DESCRIPTION
**Pull request by apdavison**
(cf #649)

Not implemented: write support for IrregularlySampledSignals

### CODE REVIEW

**Comment by apdavison** [comment]
anyone feel like merging this?

File: neo/io/asciisignalio.py [hunk 1]
```diff
@@ -228,7 +228,13 @@ def read_segment(self, lazy=False):
             # todo: if the values in timecolumn are not equally spaced
```
Comment (JuliaSprenger) [inline neo/io/asciisignalio.py]: this comment can be removed now, right?

Comment (apdavison) [reply neo/io/asciisignalio.py]: done!

**Comment by JuliaSprenger** [comment]
Looks good to me. I will merge this.

**Pull request merged**